### PR TITLE
add gha / remove travis

### PR DESCRIPTION
**Pull request by cutz**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+env:
+  PYTHONHASHSEED: 1042466059
+
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [2.7, pypy2, pypy3, 3.6, 3.7, 3.8, 3.9]
```
Comment (jamadden) [inline .github/workflows/tests.yml:13]: These same versions need listed in the classifiers in setup.py and in the interpreter list in tox.ini. Plus a note in CHANGES.rst about the expanded support.

**Review by jamadden — APPROVED** [review]
LGTM. 

The failure is due to a "drop" in coverage that can be solved with a `#pragma: no cover` on the `ImportError` line for ZODB.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (cutz) [reply .github/workflows/tests.yml:13]: As we are only expanding support is a minor version bump fine?

Comment (jamadden) [reply .github/workflows/tests.yml:13]: I think so, that’s what zope does.

**Pull request merged**